app/experimenter/nimbus-ui/src/setupTests.ts

```diff
@@ -2,4 +2,5 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import "mutationobserver-shim";
```

Comment:
Note to self / anyone else looking, this let's us use `useForm`.

Reply:
Yeah, I ran into a MutationObserver error and found this in the react-hook-form FAQ after some googling:

https://react-hook-form.com/faqs#TestingfailedduetoMutationObserver